.github/workflows/custom-actions/build-player/action.yaml

```diff
@@ -0,0 +1,14 @@
+name: Build Shaka Player
+
+description: A reusable action to build Shaka Player.
```

Comment:
Given how simple this action is for Player (one line in bash to call a script we already wrote), I think we should do away with the custom action and inline it into the workflow.  Same for linting and testing.